src/hooks/useSubscriptions.ts

```diff
@@ -110,8 +107,8 @@ export function useProposalSubscription(
   const [proposal, setProposal] = useState(null)
   const [loading, setLoading] = useState(true)
 
-  const rawProposalRef = useRef<unknown>(null)
-  const proposalSubscription = useRef<unknown>(null)
+  const rawProposalRef = useRef<any>(null)
+  const proposalSubscription = useRef<any>(null)
```

Comment:
why did we change the type from `unknown` to `any`? TS recommends to use `unknown` when you don't know what are the types of the attributes.

So, can we revert it to `unknown` like it was earlier?

Reply:
Done ✅